src/openforms/submissions/tests/form_logic/test_modify_components.py

```diff
@@ -861,7 +862,7 @@ def test_response_contains_submission(self):
             },
             actions=[
                 {
-                    "form_step": str(form_step2.uuid),
+                    "form_step": f"http://example.com{reverse('api:form-steps-detail', kwargs={'form_uuid_or_slug': form.uuid, 'uuid': form_step2.uuid})}",
```

Comment:
Next time, please don't do the `reverse` call inside the f-string, it's very unreadable code